<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblZoom">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Datei">
    <property name="title">
     <string>&amp;Datei</string>
    </property>
    <widget class="QMenu" name="menu_Drucken">
     <property name="title">
      <string>&amp;Drucken</string>
     </property>
     <addaction name="action_Vorschau"/>
     <addaction name="action_Drucken_2"/>
    </widget>
    <addaction name="action_Neu"/>
    <addaction name="action_Oeffnen"/>
    <addaction name="action_Speichern"/>
    <addaction name="actionSpeichern_unter"/>
    <addaction name="separator"/>
    <addaction name="menu_Drucken"/>
    <addaction name="separator"/>
    <addaction name="action_Beenden"/>
   </widget>
   <widget class="QMenu" name="menu_Bearbeiten">
    <property name="title">
     <string>&amp;Bearbeiten</string>
    </property>
    <addaction name="action_Rueckgaengig"/>
    <addaction name="action_Wiederholen"/>
    <addaction name="separator"/>
    <addaction name="action_Ausschneiden"/>
    <addaction name="action_Kopieren"/>
    <addaction name="action_Einfuegen"/>
    <addaction name="separator"/>
    <addaction name="action_Suchen"/>
    <addaction name="actionErsetzen"/>
   </widget>
   <widget class="QMenu" name="menu_Ansicht">
    <property name="title">
     <string>&amp;Ansicht</string>
    </property>
    <widget class="QMenu" name="menuAusrichtung">
     <property name="title">
      <string>Ausrichtung</string>
     </property>
     <addaction name="actionLinksbuendig"/>
     <addaction name="actionZentriert"/>
     <addaction name="actionRechtsbuendig"/>
    </widget>
    <widget class="QMenu" name="menu_Zoom">
     <property name="title">
      <string>&amp;Zoom</string>
     </property>
     <addaction name="actionZoomIn"/>
     <addaction name="actionZoomOut"/>
    </widget>
    <addaction name="menu_Zoom"/>
    <addaction name="action_Schriftart"/>
    <addaction name="actionFarbe"/>
    <addaction name="menuAusrichtung"/>
   </widget>
   <widget class="QMenu" name="menu_Extras">
    <property name="title">
     <string>&amp;Extras</string>
    </property>
    <widget class="QMenu" name="menu_Sprache">
     <property name="title">
      <string>&amp;Sprache</string>
     </property>
     <addaction name="actionDeutsch"/>
     <addaction name="actionEnglisch"/>
     <addaction name="actionFranz_sisch"/>
     <addaction name="actionSpanisch"/>
     <addaction name="actionRussisch"/>
    </widget>
    <addaction name="menu_Sprache"/>
    <addaction name="action_Optionen"/>
   </widget>
   <addaction name="menu_Datei"/>
   <addaction name="menu_Bearbeiten"/>
   <addaction name="menu_Ansicht"/>
   <addaction name="menu_Extras"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Neu"/>
   <addaction name="action_Oeffnen"/>
   <addaction name="action_Speichern"/>
   <addaction name="separator"/>
   <addaction name="action_Drucken_2"/>
   <addaction name="separator"/>
   <addaction name="action_Rueckgaengig"/>
   <addaction name="action_Wiederholen"/>
   <addaction name="separator"/>
   <addaction name="action_Ausschneiden"/>
   <addaction name="action_Kopieren"/>
   <addaction name="action_Einfuegen"/>
   <addaction name="separator"/>
   <addaction name="actionLinksbuendig"/>
   <addaction name="actionZentriert"/>
   <addaction name="actionRechtsbuendig"/>
   <addaction name="separator"/>
   <addaction name="action_Suchen"/>
   <addaction name="actionErsetzen"/>
   <addaction name="action_Schriftart"/>
   <addaction name="actionFarbe"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
  </widget>
  <action name="action_Neu">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/new.png</normaloff>:/Icons/Icons/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Neu</string>
   </property>
  </action>
  <action name="action_Oeffnen">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/open.png</normaloff>:/Icons/Icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Ö&amp;ffnen</string>
   </property>
  </action>
  <action name="action_Speichern">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/save.png</normaloff>:/Icons/Icons/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Speichern</string>
   </property>
  </action>
  <action name="actionSpeichern_unter">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/save.png</normaloff>:/Icons/Icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Speichern &amp;unter</string>
   </property>
  </action>
  <action name="action_Vorschau">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/find.png</normaloff>:/Icons/Icons/find.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Vorschau</string>
   </property>
  </action>
  <action name="action_Drucken_2">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/print.png</normaloff>:/Icons/Icons/print.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Drucken</string>
   </property>
  </action>
  <action name="action_Beenden">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/icons8-close-50.png</normaloff>:/Icons/Icons/icons8-close-50.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Beenden</string>
   </property>
  </action>
  <action name="action_Rueckgaengig">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/undo.png</normaloff>:/Icons/Icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rückgängig</string>
   </property>
  </action>
  <action name="action_Wiederholen">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/redo.png</normaloff>:/Icons/Icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Wiederholen</string>
   </property>
  </action>
  <action name="action_Ausschneiden">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/cut.png</normaloff>:/Icons/Icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Ausschneiden</string>
   </property>
  </action>
  <action name="action_Kopieren">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/copy.png</normaloff>:/Icons/Icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Kopieren</string>
   </property>
  </action>
  <action name="action_Einfuegen">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/paste.png</normaloff>:/Icons/Icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Einfügen</string>
   </property>
  </action>
  <action name="action_Suchen">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/find.png</normaloff>:/Icons/Icons/find.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Suchen</string>
   </property>
  </action>
  <action name="actionErsetzen">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/replace.png</normaloff>:/Icons/Icons/replace.png</iconset>
   </property>
   <property name="text">
    <string>Ersetzen</string>
   </property>
  </action>
  <action name="action_Schriftart">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/font.png</normaloff>:/Icons/Icons/font.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Schriftart</string>
   </property>
  </action>
  <action name="actionFarbe">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/color.png</normaloff>:/Icons/Icons/color.png</iconset>
   </property>
   <property name="text">
    <string>Farbe</string>
   </property>
  </action>
  <action name="actionLinksbuendig">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/align_left.png</normaloff>:/Icons/Icons/align_left.png</iconset>
   </property>
   <property name="text">
    <string>Linksbündig</string>
   </property>
  </action>
  <action name="actionZentriert">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/align_center.png</normaloff>:/Icons/Icons/align_center.png</iconset>
   </property>
   <property name="text">
    <string>Zentriert</string>
   </property>
  </action>
  <action name="actionRechtsbuendig">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/align_right.png</normaloff>:/Icons/Icons/align_right.png</iconset>
   </property>
   <property name="text">
    <string>Rechtsbündig</string>
   </property>
  </action>
  <action name="action_Optionen">
   <property name="text">
    <string>&amp;Optionen</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/zoom_in.png</normaloff>:/Icons/Icons/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/icons8-zoom-out-60.png</normaloff>:/Icons/Icons/icons8-zoom-out-60.png</iconset>
   </property>
   <property name="text">
    <string>ZoomOut</string>
   </property>
  </action>
  <action name="actionDeutsch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Deutsch</string>
   </property>
  </action>
  <action name="actionEnglisch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Englisch</string>
   </property>
  </action>
  <action name="actionFranz_sisch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Französisch</string>
   </property>
  </action>
  <action name="actionSpanisch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Spanisch</string>
   </property>
  </action>
  <action name="actionRussisch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Russisch</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
